specification/2.0/Specification.adoc

```diff
@@ -91,45 +91,168 @@ for identification and belong to their respective owners.
+This document, referred to as the “glTF Specification” or just the “Specification” hereafter, describes the glTF file format.
 
+glTF is an API-neutral runtime asset delivery format.  glTF bridges the gap between 3D content creation tools and modern graphics applications by providing an efficient, extensible, interoperable format for the transmission and loading of 3D content.
 
-[[motivation]]
-== Motivation
 
-*This section is non-normative.*
+[[introduction-conventions]]
+== Document Conventions
+
+The glTF Specification is intended for use by both implementors of the asset exporters or converters (e.g. DCC tools) and application developers seeking to import or load glTF assets, forming a contract between these parties.
+
+Specification text may address either party; typically the intended audience can be inferred from context, though some sections are defined to address only one of these parties.
+
+Any requirements, prohibitions, recommendations or options defined by <<introduction-normative-terminology, normative terminology>> are imposed only on the audience of that text.
+
+
+[[introduction-informative-language]]
+=== Informative Language
+
+Some language in the specification is purely informative, intended to give background or suggestions to implementors or developers.
+
+If an entire chapter or section contains only informative language, its title will be suffixed with "`(Informative)`".
+
+All NOTEs are implicitly informative.
+
+
+[[introduction-normative-terminology]]
+=== Normative Terminology
+
+The key words **MUST**, **MUST NOT**, **REQUIRED**, **SHALL**, **SHALL NOT**, **SHOULD**, **SHOULD NOT**, **RECOMMENDED**,  **MAY**, and **OPTIONAL** in this document are to be interpreted as described in https://tools.ietf.org/rfc/bcp/bcp14.html[BCP 14 - Key words for use in RFCs to Indicate Requirement Levels].
+
+These key words are highlighted in the specification for clarity.
+
+Unless otherwise noted in the section heading, all sections and appendices in this document are normative.
+
+
+[[introduction-technical-terminology]]
+=== Technical Terminology
+
+The glTF Specification makes use of common engineering and graphics terms such as *Vertex*, *Normal*, *Texture*, etc to identify and describe certain GPU constructs and their attributes, states, and behaviors.
+
+[[introduction-normative-references]]
+=== Normative References
+
+References to external documents are considered normative references if the Specification uses any of the normative terms defined in <<introduction-normative-terminology>> to refer to them or their requirements, either as a whole or in part.
+
+The following documents are referenced by normative sections of the specification:
+
+==== External Specifications
```

Comment:
In rendered form it's difficult to find something quickly among these references, because the keywords are somewhere in the middle of the line. Would it be appropriate to put the title first, rather than the source, or prefix with a keyword like "_JSON_"?